File: examples/python/plans_example.py
import qoery

configuration = qoery.Configuration()
configuration.api_key['ApiKeyAuth'] = 'YOUR_API_KEY'

with qoery.ApiClient(configuration) as api_client:
    api = qoery.PlansApi(api_client)
    plans = api.list_plans()
    print(f"Plans: {list(plans.plans.keys())}")

File: examples/js/plans_example.js
const Qoery = require('qoery');

const client = Qoery.ApiClient.instance;
client.authentications['ApiKeyAuth'].apiKey = 'YOUR_API_KEY';

const api = new Qoery.PlansApi();

api.listPlans((error, plans) => {
  if (error) throw error;
  console.log(`Plans: ${Object.keys(plans.plans).join(', ')}`);
});
